src/api/teams/controllers/update-team.js

```diff
@@ -58,6 +63,27 @@ const updateTeamController = {
       if (!gitHubExists) {
         throw Boom.badData('Team does not exist in GitHub')
       }
+      // This is still experimental, so we're just going to log the error from this bit for now.
+      if (
+        existingTeam?.github &&
+        (await getTeamsCount(request.db, { github: existingTeam.github })) === 1
```

Comment:
If we are to use this could we abstract this to a `teamHasPermissions`, `teamIsCollaborator` method? To make it easier to read?